_posts/2020-08-12-DDoS-criminal-hackers.md

```diff
@@ -0,0 +1,68 @@
+fusion centers and police departments and published the release in what it
+billed the #BlueLeaks release.
+
+Journalists and researchers immediately seized on the Blue Leaks release to
+reveal that police were obsessed with left-wing movements and ignoring
+far-right [domestic extremism](https://theintercept.com/2020/07/15/george-floyd-protests-police-far-right-antifa/).
+Blue Leaks further revealed that law enforcement inflated threats after the
+police murder of George Floyd in Minneapolis with some police stating, without evidence, that "Antifa"
+would carry out [car bombings](https://theintercept.com/2020/06/26/blueleaks-minneapolis-police-protest-fears/).
+The shoddy intelligence of fusion centers has been [well documented](https://www.hsgac.senate.gov/imo/media/doc/10-3-2012%20PSI%20STAFF%20REPORT%20re%20FUSION%20CENTERS.2.pdf)
+for over fifteen years.  
+
+Nonetheless, authorities moved swiftly to crack down on DDoS's right to publish, first by [seizing its server](https://www.infosecurity-magazine.com/news/german-police-seize-blueleaks/)
+in the middle of the night presumably on behalf of the FBI. Homeland Security
+Investigations (HSI), one of the largest agencies within DHS, visited
+unaffiliated individuals hosting an archives of the Blue Leaks and even [offered them money](https://www.vice.com/en_us/article/xg84ww/ice-questions-admin-of-the-eye-archive-site-hosted-blue-leaks) to become informants. Just this week, documents obtained by Lucy Parsons Labs [confirm](https://cdn.muckrock.com/foia_files/2020/08/10/CCSO_00000003.pdf) DHS considers DDoS "criminal hackers" and not journalists.
+
+
+The government's aggressive push to name DDoS as "criminal hackers" is a
+dangerously unprecedented territory that criminalized everyone's right to
+publish online under the First Amendment. It is not illegal to possess and
+publish information obtained by a third party, even if it was obtained
+illegally, under the First Amendment as long as an individual does not
+participate in the illegal activity. This extends even to the publishing of
+classified information, as the Supreme Court
+[upheld](https://en.wikipedia.org/wiki/New_York_Times_Co._v._United_States) in
+the infamous case of the Pentagon Papers. (Almost all of the data published in
+the Blue Leaks was
+[unclassified](https://www.dla.mil/Portals/104/Documents/InformationOperations/LogisticsInformationServices/Customer%20Outreach/Subscriptions/J6_ForOfficialUseOnly_160113.pdf).)
+
+The confirmation of an investigation by DHS's Office of Intelligence and
+Analysis (I&A) reflect a growing patter of DHS investigating journalists as
+threats to national security. Just this month in Portland, I&A created "baseball cards" of
+[journalists](https://www.politico.com/news/2020/08/01/dhs-removes-official-who-oversaw-intelligence-reports-on-journalists-390042),
+a term most commonly used by the military in hostile conflict situations. Other DHS
```

Comment:
Maybe "practice" instead of "term"? I believe this word is referring to the baseball cards.